sql/src/test/java/ddlog/JooqProviderTestBase.java

```diff
@@ -539,6 +539,34 @@ public void testInsertNullFails1() {
         }
     }
 
+    @Test
+    public void testArrayAggTypes() {
+        assert(create != null);
+        create.execute("insert into \nhosts values ('n1', 10, true)");
+        create.batch("insert into hosts values ('n54', 18, false)",
```

Comment:
none of your tests has an array with more than 1 element, or with a null element.
is it possible to add these?

Reply:
Added both types of array results.